[new-product] bun (#4382)
captn3m0/endoflife.date: products/bun.md

---
title: Bun
category: framework

> tags: javascript-runtime

Reviewer: I see `nodejs` is also tagged with this, but I was considering `javascript-runtime` as products "that run on the JS runtime". 

@marcwrobel thoughts?

Reviewer 2: That's correct, I had this in mind too.

A similar case is Java, for which the various distribution has been flagged with https://endoflife.date/tags/java-distribution. But in this case it is in fact really a JavaScript runtime, so not sure what to do about it. So let's use the same tags as Node.js for now et leave it like that.

iconSlug: bun
permalink: /bun
versionCommand: bun --version
releasePolicyLink: https://github.com/oven-sh/bun/releases
changelogTemplate: 
  https://bun.sh/blog/bun-v__LATEST__
releaseDateColumn: true

identifiers:
-   purl: pkg:docker/oven/bun

> -   repology: bun

Reviewer: Added a purl directly to github as well.

```suggestion
-   repology: bun
-   purl: pkg:github/oven/bun
```

auto:
-   git: https://github.com/oven-sh/bun

releases:
-   releaseCycle: "1"
    releaseDate: 2023-09-08
    eol: false

>     latest: "1.0.20"
>     latestReleaseDate: 2023-12-24

Author: ```suggestion
    latest: "1.0.21"
    latestReleaseDate: 2024-01-02
```

---

> > [Bun](https://bun.sh/) is a open-source JavaScript all-in-one toolkit
> 
> Develop, test, run, and bundle JavaScript & TypeScript projects—all with Bun.
> Bun is an all-in-one JavaScript runtime & toolkit designed for speed, complete with a bundler, test runner, and Node.js-compatible package manager.

Reviewer: ```suggestion
> [Bun](https://bun.sh/) is a open-source JavaScript runtime that focuses on speed,
and comes with a bundler, test runner, and a Node.js-compatible package manager.

Bun does not have a clearly defined support policy.
```

---
title: Bun
category: framework
tags: javascript-runtime
iconSlug: bun
permalink: /bun
versionCommand: bun --version
releasePolicyLink: https://github.com/oven-sh/bun/releases
changelogTemplate: 
  https://bun.sh/blog/bun-v__LATEST__
releaseDateColumn: true

identifiers:
-   purl: pkg:docker/oven/bun
-   repology: bun
-   purl: pkg:github/oven/bun

auto:
-   git: https://github.com/oven-sh/bun

releases:
-   releaseCycle: "1"
    releaseDate: 2023-09-08
    eol: false

>     latest: "1.0.21"
>     latestReleaseDate: 2024-01-02

Author: ```suggestion
    latest: "1.0.26"
    latestReleaseDate: 2024-02-03
```
